Controller/MediaController.php

```diff
@@ -134,12 +134,25 @@ public function liipImagineFilterAction($path, $filter)
 
         $image = $this->get('liip_imagine')->open($tmpFile);
 
-        $response = $this->get('liip_imagine.filter.manager')->get($this->get('request'), $filter, $image, $path);
+        $response = $this->get('liip_imagine.filter.manager')->get($this->getRequest(), $filter, $image, $path);
 
         if ($targetPath) {
             $response = $this->get('liip_imagine.cache.manager')->store($response, $targetPath, $filter);
         }
 
         return $response;
     }
+    
+    
+        /**
+     * @return Request
+     */
+    public function getRequest()
```

Comment:
Please add a `NEXT_MAJOR` comment explaining what needs to be done on next major.